autoscale/src/test/java/com/sequenceiq/periscope/service/YarnBasedScalingAdjustmentServiceTest.java

```diff
@@ -433,6 +437,7 @@ private void setupBasicMocks(StackV4Response stackResponse, YarnScalingServiceV1
         doReturn(yarnResponse).when(yarnMetricsClient).getYarnMetricsForCluster(any(Cluster.class), eq(stackResponse), eq(HOSTGROUP), eq(POLLING_USER_CRN),
                 any(Optional.class));
         doReturn(TEST_MESSAGE).when(messagesService).getMessageWithArgs(anyString(), any());
+        doNothing().when(metricService).recordYarnInvocation(any(Cluster.class), anyLong());
```

Comment:
Can we verify some of the expected metrics being populated ?

Reply:
Added verification